docs/source/getting-started.rst

```diff
@@ -1,10 +1,10 @@
-===============
-Getting Started
-===============
+====================================================
+Getting Started with SymbiFlow Toolchain development
+====================================================
 
 .. note::
```

Comment:
Might be worth changing this to a warning?

Reply:
That's a good idea - updated